routemodel/maps.py

```diff
@@ -0,0 +1,212 @@
+import requests
+import pandas as pd
+
+base_url = 'https://dev.virtualearth.net/REST/v1/'
```

Comment:
if this is constant, should be ALL_CAPS. BASE_URL

Reply:
Resolved.